src/io/ble.js

```diff
@@ -172,7 +174,10 @@ class BLE extends JSONRPCWebSocket {
     _sendError (/* e */) {
         this.disconnect();
         // log.error(`BLE error: ${JSON.stringify(e)}`);
-        this._runtime.emit(this._runtime.constructor.PERIPHERAL_ERROR);
+        this._runtime.emit(this._runtime.constructor.PERIPHERAL_ERROR, {
```

Comment:
Maybe we should just have an API for this, so we can just use `this._runtime.emitPeripheralError(this._extensionId)`?

Reply:
An API like this sounds good, maybe for another pass that includes localization, like you said?